.env

```diff
@@ -1,6 +1,10 @@
+PORT=3000
+WDS_SOCKET_PORT=3000
 PUBLIC_URL=/
-REACT_APP_PAGE_MODEL_PATH=/content/wknd-spa-react/us/en.model.json
+BUILD_PATH=dist/
+INLINE_RUNTIME_CHUNK=false
 REACT_APP_API_HOST=http://localhost:4502
 REACT_APP_AEM_AUTHORIZATION_HEADER='Basic YWRtaW46YWRtaW4='
-REACT_APP_ROOT=/content/wknd-spa-react/us/en/home.html
+REACT_APP_ROOT=/content/aem-react-spa/language-masters/en/home.html
```

Comment:
Since before there wasn't this "language-masters", I suppose the archetype that you created now generates this "language-masters" path?